docs/headless-form/changeset/demo/base-demo.md

```diff
@@ -0,0 +1,58 @@
+```hbs template
+{{! note - write your own map-error helper }}
+{{! field.rawErrors.map(error => error.message) }}
```

Comment:
Another option would maybe be to add a local helper instead so that it sits with the existing code here in the demo example rather than being in our docs app